lib/ohai/plugins/cloud.rb

```diff
@@ -189,7 +189,7 @@ def get_openstack_values
     cloud[:public_hostname] = openstack["public_hostname"]
     cloud[:local_ipv4] = openstack["local_ipv4"]
     cloud[:local_hostname] = openstack["local_hostname"]
-    cloud[:provider] = openstack["provider"]
+    cloud[:provider] = "openstack"
```

Comment:
So this one gets a bit weird and here's why I went with openstack. When you're in the context of the cloud plugin says provider you'd think openstack (right? maybe...). When you're in the context of the openstack plugin you know you're openstack so when someone says provider you're thinking about what openstack provider and that could be dreamhost. I guess it could really go either way though.